flow/core/kernel/kernel.py

```diff
@@ -1,7 +1,7 @@
 """Script containing the Flow kernel object for interacting with simulators."""
 
 from flow.core.kernel.simulation import TraCISimulation
-from flow.core.kernel.scenario import KernelScenario
+from flow.core.kernel.scenario import TraCIScenario
```

Comment:
A bit confused by this file structure overall. Will kernel.scenario contain both TraciScenario and AimsunScenario?

Reply:
yup! AimsunScenario will be in a later PR though